### ISSUE TITLE
Format CSV - Encoding Issues - Accents French Language not recognized in csv exported file 

### ISSUE THREAD

**Issue by Alexo47** [GateNLP/gateplugin-Format_CSV#1]
Hello!

When I use Format CSV "Save As" (export with encoding parameter = UTF-8) the csv file produced does not encode properly the accents (french) and my data is crippled.

I tried Notepad++ to transform the character set but was not successful.

=> Any help to correct this issue will be appreciated.

Cordially,
Alexandre

**Comment by ianroberts** [comment]
Please provide an example input document and the settings you are using with the CSV exporter to enable us to reproduce the bug.

**Comment by ianroberts** [comment]
[Example input file from gate-users message](https://github.com/GateNLP/gateplugin-Format_CSV/files/6503734/TEST-FRA_Suetone-Auguste.docx)

Exporter settings:

![GATE-OCT_CSV-Settings](https://user-images.githubusercontent.com/428107/118710249-f342d700-b815-11eb-9fb8-4c705b87be93.jpg)

**Comment by Alexo47** [comment]
Thank you for your interest in this issue - Please find attached the following files:

    Image of the PR Pipeline (only my project related Gazetteer is active)
    The Word doc selected for analysis
    The CSV dialog box
    The xlsx file with accents crippled (the git-hub l attachr efuses my exported CSV file, don't understand why - just saved in xlsx format)

Cordially,
Alexandre


![GATE-OCT_PipelineProcessors](https://user-images.githubusercontent.com/76005554/118710508-add6d780-b81e-11eb-907e-0947d5ffac8f.jpg)
[TEST-FRA_Suetone-Auguste.docx](https://github.com/GateNLP/gateplugin-Format_CSV/files/6503759/TEST-FRA_Suetone-Auguste.docx)
![GATE-OCT_CSV-Settings](https://user-images.githubusercontent.com/76005554/118710548-bd562080-b81e-11eb-8022-fba6872ee61f.jpg)

[TEST-FRA_Auguste-Agrippa_RELATION.xlsx](https://github.com/GateNLP/gateplugin-Format_CSV/files/6503782/TEST-FRA_Auguste-Agrippa_RELATION.xlsx)

**Comment by Alexo47** [comment]
I was not able to load the exported CSV file in the git-hub messaging 
protocol

Please find attached my CSV file

Cordially,
Alexandre

------ Original Message ------
From: "Ian Roberts" ***@***.***>
To: "GateNLP/gateplugin-Format_CSV" 
***@***.***>
Cc: "Alexo47" ***@***.***>; "Author" ***@***.***>
Sent: 18/05/2021 21:17:08
Subject: Re: [GateNLP/gateplugin-Format_CSV] Format CSV - Encoding 
Issues - Accents French Language not recognized in csv exported file 
(#1)

**Comment by ianroberts** [comment]
This is not a bug in the CSV exporter, the problem is down to how Excel handles opening of CSV files.  The output CSV you supplied on the gate-users list was correct UTF-8, but if you just double click on the file then Excel will open it assuming it is encoded in a single-byte Windows character set, resulting in the corruption you see.  If you create a blank Excel workbook and then use the _import_ function to load the CSV data you can select "Unicode (UTF-8)" as the file format (or "origin" as it calls it on my Mac version), and the accented characters load correctly.

![Screenshot 2021-05-18 at 20 31 01](https://user-images.githubusercontent.com/428107/118711910-09519700-b818-11eb-8d92-82915908d35d.png)

**Comment by ianroberts** [comment]
Alternatively you could set "windows-1252" or "ISO-8859-15" as the encoding for the CSV exporter, one of those should produce a CSV file in a single-byte encoding that Excel can open with its default settings.

**Comment by Alexo47** [comment]
Hello Ian,

Thanks for your feedback.

I tried excel Data-Import and  had to change the indicated file origin encoding (1252-Windows) to UTF-8 and then I got  the proper data with the correct representation of the accents.

I appreciate your support.

Cordially,
Alexandre

![Excel_GetData_Transformed-UTF8](https://user-images.githubusercontent.com/76005554/118715821-63585980-b824-11eb-8ad0-53d69333bf26.jpg)



![Excel_GetData_1252-Windows-Encoding](https://user-images.githubusercontent.com/76005554/118715142-90f0d300-b823-11eb-9d6e-138830542411.jpg)
